internal/v4/api.go

```diff
@@ -222,6 +223,37 @@ func (h *Handler) updateEntity(id *charm.Reference, fields map[string]interface{
 	return nil
 }
 
+func (h *Handler) updateSearch(id *charm.Reference, fields map[string]interface{}) error {
```

Comment:
I don't think it is actually used, it is provided to the Field updater should it need it.